src/20250426_Multi_MayFundingUpdate/AaveV3Ethereum_MayFundingUpdate_20250426.sol

```diff
@@ -0,0 +1,195 @@
+      MiscEthereum.ECOSYSTEM_RESERVE,
+      IERC20(AaveV3EthereumAssets.AAVE_UNDERLYING).balanceOf(COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.transfer(
+      IERC20(AaveV3Ethereum.COLLECTOR.ETH_MOCK_ADDRESS()),
+      ACI,
+      ETH_TRANSFER_AMOUNT
+    );
+  }
+
+  function _swaps() internal {
+    AaveV3Ethereum.COLLECTOR.swap(
+      MiscEthereum.AAVE_SWAPPER,
+      CollectorUtils.SwapInput({
+        milkman: MILKMAN,
+        priceChecker: PRICE_CHECKER,
+        fromUnderlying: AaveV3EthereumAssets.USDT_UNDERLYING,
+        toUnderlying: AaveV3EthereumAssets.GHO_UNDERLYING,
+        fromUnderlyingPriceFeed: AaveV3EthereumAssets.USDT_ORACLE,
+        toUnderlyingPriceFeed: GHO_USD_FEED,
+        amount: USDT_SWAP_AMOUNT,
+        slippage: 75
+      })
+    );
+
+    AaveV3Ethereum.COLLECTOR.swap(
+      MiscEthereum.AAVE_SWAPPER,
+      CollectorUtils.SwapInput({
+        milkman: MILKMAN,
+        priceChecker: PRICE_CHECKER,
+        fromUnderlying: AaveV3EthereumAssets.USDC_UNDERLYING,
+        toUnderlying: AaveV3EthereumAssets.GHO_UNDERLYING,
+        fromUnderlyingPriceFeed: AaveV3EthereumAssets.USDC_ORACLE,
+        toUnderlyingPriceFeed: GHO_USD_FEED,
+        amount: USDC_SWAP_AMOUNT,
+        slippage: 75
+      })
+    );
+
+    /// ~ 2.3M
+    AaveV3Ethereum.COLLECTOR.swap(
+      MiscEthereum.AAVE_SWAPPER,
+      CollectorUtils.SwapInput({
+        milkman: MILKMAN,
+        priceChecker: PRICE_CHECKER,
+        fromUnderlying: AaveV3EthereumAssets.DAI_UNDERLYING,
+        toUnderlying: AaveV3EthereumAssets.USDS_UNDERLYING,
+        fromUnderlyingPriceFeed: AaveV3EthereumAssets.DAI_ORACLE,
+        toUnderlyingPriceFeed: AaveV3EthereumAssets.USDS_ORACLE,
+        amount: IERC20(AaveV3EthereumAssets.DAI_UNDERLYING).balanceOf(COLLECTOR),
+        slippage: 75
+      })
+    );
+
+    /// ~7k
+    AaveV3Ethereum.COLLECTOR.swap(
+      MiscEthereum.AAVE_SWAPPER,
+      CollectorUtils.SwapInput({
+        milkman: MILKMAN,
+        priceChecker: PRICE_CHECKER,
+        fromUnderlying: AaveV3EthereumAssets.LUSD_UNDERLYING,
+        toUnderlying: AaveV3EthereumAssets.WETH_UNDERLYING,
+        fromUnderlyingPriceFeed: AaveV3EthereumAssets.LUSD_ORACLE,
+        toUnderlyingPriceFeed: AaveV3EthereumAssets.WETH_ORACLE,
+        amount: IERC20(AaveV3EthereumAssets.LUSD_UNDERLYING).balanceOf(COLLECTOR),
+        slippage: 350
+      })
+    );
+
+    /// ~20k
+    AaveV3Ethereum.COLLECTOR.swap(
+      MiscEthereum.AAVE_SWAPPER,
+      CollectorUtils.SwapInput({
+        milkman: MILKMAN,
+        priceChecker: PRICE_CHECKER,
+        fromUnderlying: AaveV2EthereumAssets.CVX_UNDERLYING,
+        toUnderlying: AaveV3EthereumAssets.WETH_UNDERLYING,
+        fromUnderlyingPriceFeed: CVX_USD_ORACLE,
+        toUnderlyingPriceFeed: AaveV3EthereumAssets.WETH_ORACLE,
+        amount: IERC20(AaveV2EthereumAssets.CVX_UNDERLYING).balanceOf(COLLECTOR),
+        slippage: 200
+      })
+    );
+  }
```

Comment:
Missing swaps for RAI and FRAX. Saw that RAI is not on the readme table, but FRAX is, so if the swap is to not be done, then should be removed from there. (Or add the necessary swaps here)